examples/erc20/DeFi/Compound/supply.en.shtml

```diff
@@ -5,6 +5,21 @@ class Token {
+            <span><bold><h3>${this.props.name} Balance: ${cTokenBalance}</h3></bold></span>
+            <span><bold><h3>${this.props.name.replace("c", "")} Balance: ${tokenBalance}</h3></bold></span>
```

Comment:
Can the `replace("c", "")` be factorised into one place rather than duplicated in multiple places?